guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -70,7 +70,7 @@ euid = __ID_of_Apache_User__
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # systemctl restart gssproxy.service
```

Comment:
The restart here is weird. Is there a specific reason for this?

Reply:
It was added in the "original" version of that doc (in https://bugzilla.redhat.com/show_bug.cgi?id=1206138) but I don't see any real reason for that, no.

However, looking at a system that I have here, which *does not have IPA enabled at all*, the `gssproxy` service is running, even if it's marked as not enabled:

```console
# systemctl status gssproxy
● gssproxy.service - GSSAPI Proxy Daemon
   Loaded: loaded (/usr/lib/systemd/system/gssproxy.service; disabled; vendor preset: disabled)
   Active: active (running) since Fri 2022-11-25 07:07:56 UTC; 1h 47min ago
 Main PID: 745 (gssproxy)
    Tasks: 6 (limit: 50696)
   Memory: 1.6M
   CGroup: /system.slice/gssproxy.service
           └─745 /usr/sbin/gssproxy -D
```

Some steps above we do edit the config, so if the service is already running, a restart is really required.

Reply:
I'm guessing something activates it. Either via a socket or something else via a requirement. Let's be safe and keep restart in then. It doesn't hurt in this order (first `enable --now` and then `restart` could cause a double service start, which is slower).

Reply:
I could not find *what* activated it, but yeah there must be something.

And agreed, it doesn't hurt that way, so let's keep it